wp-cli.php

```diff
@@ -379,6 +379,10 @@ public function ip_list( $args, $assoc_args ) {
 	 *    $ wp rsa ip-add 192.0.0.1
 	 *    Success: Added 192.0.0.1 to site whitelist.
 	 *
+	 *    # Adds 8.8.8.8 1.1.1.1 with labels Google and Cloudflare.
+	 *    $ wp rsa ip-add 8.8.8.8=Google 1.1.1.1=Cloudflare
```

Comment:
Using `=` makes the IP look like a flag while it's not. Personally, I prefer `|` or `,`. But I think we should do a quick survey (in our engineering channel).

Also, from the code, look like we haven't support spaces in labels. We should support it or note the limitation in the docs.

Reply:
Would it make more sense to have the label be a parametered property? 

```
wp rsa ip-add 8.8.8.8 --label=Google
```

Reply:
@mikelking it makes sense if we add only one IP, but the `ip-add`command supports adding multiple IPs at a time.

Reply:
> Also, from the code, look like we haven't support spaces in labels. We should support it or note the limitation in the docs.

@dinhtungdu Labels work like any other WP-CLI command. To add spaces, enclose the label in double quotes.
For example:
```
wp rsa ip-add 8.8.8.8=Google 8.8.8.8.="Google alternate" 10.20.1.2 4.4.4.4="Cloudflare DNS"
```

Reply:
@dinhtungdu 

I understand your point it just doesn't feel very consistent with WPCLI in general. In either case will this multiple IP=label construct function with ip scopes? 

```
wp rsa ip-add 2a06:98c0::/29="Cloudflare 6" 2c0f:f248::/32="Cloudflare 7"
```

Reply:
@mikelking Yes, that's the current command to add multiple IP with labels. I know it's inconsistent and I'm thinking about a better way to structure it. I'm wondering if the bellow is possible:
```sh
wp rsa ip-add 8.8.8.8 --label="Google" 1.1.1.1 --label="Cloudflare"
```
cc @Sidsector9

Reply:
@dinhtungdu flags have to be unique in a WP-CLI command as they form associative array keys for the `$assoc_args` argument. Personally, I feel this syntax will be tedious for the end user. IMO `|` would be a better choice in comparison.

@mikelking The command will work with IP scopes.

Reply:
```
wp rsa ip-add 8.8.8.8|"Google with spaces"
```

@Sidsector9 With spaces and quotation marks, `=` is better than `|` to me.